### ISSUE TITLE
Cannot import package, incorrect exports

### ISSUE THREAD

**Issue by pgiraud** [juanjoDiaz/streamparser-json#19]
I tried to use the package in a vite project and I get the following error:

```
[vite] Internal server error: Failed to resolve entry for package "@streamparser/json". The package may have incorrect main/module/exports specified in its package.json.
```

It seems like the "module" key in `package.json` points to a not existing file `./dist/mjs/index.js`.

**Comment by tobiasfuhlroth** [comment]
@pgiraud Had the same problem. See https://github.com/juanjoDiaz/streamparser-json/pull/18 for a solution.

**Comment by juanjoDiaz** [comment]
Thanks for reporting!
Solved in version 0.10
